ox-jira.el

```diff
@@ -194,6 +194,12 @@ CONTENTS is nil.  INFO is a plist used as a communication
 channel."
   (format "{{%s}}" (org-element-property :value code)))
 
+(defun ox-jira-entity (entity _contents info)
+  "Transcode a CODE object from Org to JIRA.
+CONTENTS is nil.  INFO is a plist used as a communication
+channel."
+  (org-element-property :utf-8 entity))
```

Comment:
🤔 What does this actually do? A test would really help here, as I don't know what this is for.